<launch>

<!--
Run as, e.g., 

    roslaunch aprs_objects aprs_objects.launch debug:=0x01
-->

<arg name="debug" default="0" />
<arg name="cam1" default="5001" />
<arg name="cam2" default="5002" />

<param name="debug" value="0" type="int" />

<param name="cam1_x_min" value="-1000" type="double" />
<param name="cam1_y_min" value="0" type="double" />
<param name="cam1_x_max" value="1000" type="double" />
<param name="cam1_y_max" value="1000" type="double" />

<param name="cam2_x_min" value="-1000" type="double" />
<param name="cam2_y_min" value="-1000" type="double" />
<param name="cam2_x_max" value="1000" type="double" />
<param name="cam2_y_max" value="0" type="double" />

<node pkg="aprs_objects" name="aprs_objects" type="aprs_objects_node" args="-d $(arg debug) -1 $(arg cam1) -2 $(arg cam2)" output="screen" />

</launch>
